src/objects.h

```diff
@@ -10381,6 +10373,7 @@ class Float32x4: public JSObject {
   // Helpers.
   static const int kLanes = 4;
   inline float getAt(int index);
+  inline float get_lane(int lane) {return getAt(lane);}
   inline float x() { return getAt(0); }
```

Comment:
If we don't use x()/y()/z()/w() getters anymore, please remove them. Ditto for Float64x2 and Int32x4.